news/4486.bugfix

```diff
@@ -0,0 +1 @@
+Improve handling of Unicode output from build tools under Python 3.
```

Comment:
I'm leaving out the phrase "improperly encoded", as it's mostly text that *is* properly encoded, just not in the same encoding that we're assuming. (There's no standard yet for what encoding setuptools should be using for output).